save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/CloneRepositoryController.kt

```diff
@@ -93,12 +93,8 @@ class CloneRepositoryController(
         log.info("Sending request to preprocessor to start save file for project id=$projectId")
         val bodyBuilder = MultipartBodyBuilder()
         bodyBuilder.part("executionRequestForStandardSuites", executionRequestForStandardSuites)
-        binaryFile.subscribe { binFile ->
-            bodyBuilder.part("binFile", binFile)
-            propertyFile.subscribe { propFile ->
-                bodyBuilder.part("property", propFile)
-            }
-        }
+        binaryFile.flatMap { Mono.just(bodyBuilder.part("binFile", it)) }
+        binaryFile.flatMap { Mono.just(bodyBuilder.part("property", it)) }
         return preprocessorWebClient
             .post()
```

Comment:
```suggestion
        return binaryFile.map { bodyBuilder.part("binFile", it) }
            .then {
                propertyFile.map { bodyBuilder.part("property", it)
            }
            .then {
                preprocessorWebClient
                    .post()
                    ...
            }
```
We can chain monos like this, I suppose. Also, even if you flatMap a mono, you'll still need to subscribe to it